src/controllers/objecting.entity.name.controller.ts

```diff
@@ -16,6 +22,67 @@ const validators = [
   check(DIVULGE_INFO_FIELD).not().isEmpty().withMessage(ErrorMessages.SELECT_TO_DIVULGE),
 ];
 
+const getSessionPageDataIfPresent = (session: Session, res: Response) => {
+  let existingName;
+  let yesChecked: boolean = false;
+  let noChecked: boolean = false;
+  const objectionCreate: ObjectionCreate = retrieveFromObjectionSession(session, SESSION_OBJECTION_CREATE);
+  if (objectionCreate) {
+    existingName = objectionCreate.fullName;
+    const existingSharedIdentity = objectionCreate.shareIdentity;
+    yesChecked = existingSharedIdentity;
+    noChecked = !existingSharedIdentity;
+  }
+
+  return res.render(Templates.OBJECTING_ENTITY_NAME, {
+    fullNameValue: existingName,
+    isYesChecked: yesChecked,
+    isNoChecked: noChecked,
+    templateName: Templates.OBJECTING_ENTITY_NAME,
+  });
+}
+
+const getExistingPageData = async (session: Session, res: Response, next: NextFunction) => {
+  try {
+    const objection: Objection = await getObjection(session) as Objection;
```

Comment:
I don't believe we need "as Objection" here, getObjection returns an Objection, and you have set the type to Objection without it.

Reply:
Remove dunecessary typing